cyclonedx/model/__init__.py

```diff
@@ -695,35 +695,35 @@ class LicenseChoice:
         See the CycloneDX Schema definition: https://cyclonedx.org/docs/1.4/xml/#type_licenseChoiceType
     """
 
-    def __init__(self, *, license_: Optional[License] = None, expression: Optional[str] = None) -> None:
-        if not license_ and not expression:
+    def __init__(self, *, license: Optional[License] = None, expression: Optional[str] = None) -> None:
```

Comment:
At this point, it would be better to have the variable (not the property) called `license_` 
because here in the scope, it would shadow pythons function `license()`.
As long as we don't call `license()` in this method scope, it is safe. 

who wwould ever call `license()`?! it prints the license of python ...

Reply:
As above